pagopa/io-app
# feat(Firma con IO): [SFEQS-1165] Add signing request with identification screen and thank you page

## Short description
In this PR has been added the signing request using the identificationScreen as unlocking path. It also has been added new FCI definitions.

https://user-images.githubusercontent.com/49342144/207458215-b3298453-fe0c-4104-87c2-d4192d6e85cd.mov

## List of changes proposed in this pull request
- Update FCI specs
- Update of some refs related to new FCI specs 
- Add `createSignature` request
- Add TYP screen
- Refactor of `fciAbortRequest`
- Update stack navigator
- Add some tests

## How to test
* if not yet merged get this branch https://github.com/pagopa/io-dev-api-server/tree/SFEQS-1082-fci-endpoints from io-dev-server-api
* enable FCI_ENABLED ff
* follow signature flow
* on the QTSP clauses screen check all clauses
* on identificationScreen type pin or use biometric to unlock the path and start signing request
